dracut/image-boot/eos-image-boot-setup

```diff
@@ -18,12 +18,30 @@ if getargbool 0 endless.live_boot; then
   kpartx_roflag=-r
 fi
 
-fstype=$(lsblk --noheadings -o FSTYPE "${host_device}")
+fstype=$(lsblk -d --noheadings -o FSTYPE "${host_device}")
 if [ $? != 0 ]; then
   echo "image-boot: failed to detect filesystem type"
   exit 1
 fi
 
+if [ "$fstype" = "iso9660" ]; then
+    loopdev="/dev/mapper/endless-image"
```

Comment:
I'm also curious why we can't just let losetup pick a loop device of its own name and number